backend/cmm_helpers.ml

```diff
@@ -2844,7 +2891,7 @@ let read_from_closure_given_machtype t clos base_offset dbg =
             load Double non_scanned_pos )
         | Vec128 ->
           ( (non_scanned_pos + ints_per_vec128, scanned_pos),
-            load Onetwentyeight non_scanned_pos )
+            load Onetwentyeight_unaligned non_scanned_pos )
```

Comment:
This should probably have a comment that vec128s in closures might be misaligned